README.md

```diff
@@ -78,6 +106,10 @@ Learn more about us:
 - [Arrow Docs](https://www.arrowair.com/docs/intro)
 - [Discord](https://discord.com/invite/arrow)
 
+## LICENSE Notice
+
+Please note that svc-cargo is under BUSL license until the Change Date, currently the earlier of two years from the release date. Exceptions to the license may be specified by Arrow Governance via Additional Use Grants, which can, for example, allow svc-template to be deployed for certain production uses. Please reach out to Arrow DAO to request a DAO vote for exceptions to the license, or to move up the Change Date.
```

Comment:
There's one more svc-template hiding lol. Other repos may have the same issue.

Reply:
oh, haha, was too fast, fixing :)